src/main/java/io/confluent/kafkarest/KafkaRestConfig.java

```diff
@@ -96,6 +96,12 @@
       "Number of threads to run produce requests on.";
   public static final String PRODUCER_THREADS_DEFAULT = "5";
 
+  public static final String AUTHENTICATION_MAPPING_CONFIG = "authentication.mapping";
```

Comment:
While I agree, I don't think there's a clear rule.
`key.serializer` is not suffix by class, nor `partition.grouper`, `value.serde` or `timestamp.extractor`.

Though it may be more explicit to add a `.class` suffix.